locales-src/newtab.ftl

```diff
@@ -36,3 +36,15 @@ newtab-topsites-url-validation = Valid URL required
 newtab-topsites-image-url-label = Custom Image URL
 newtab-topsites-use-image-link = Use a custom image…
 newtab-topsites-image-validation = Image failed to load. Try a different URL.
+
+newtab-topsites-cancel-button = Cancel
+newtab-topsites-delete-history-button = Delete from History
+newtab-topsites-save-button = Save
+newtab-topsites-preview-button = Preview
+newtab-topsites-add-button = Add
+
+newtab-confirm-delete-history-p1 = Are you sure you want to delete every instance of this page from your history?
+# p1 and p2 are in the same delete dialog. "This action" refers to deleting a page from history.
```

Comment:
p1 and p2 can be confusing,  I recommend updating comment here to 
# LOCALIZATION NOTE (newtab-confirm-delete-history-p2): this string is displayed in
# the same dialog as newtab-confirm-delete-history-p1. "This action" refers to deleting a page from history.

Reply:
It seems we are flexible and not using [LOCALIZATION NOTE (<string id>)](https://searchfox.org/mozilla-central/source/browser/locales/en-US/browser/preferences/preferences.ftl#85) in fluent files and should be ok changing comment to use complete string id instead of p1 and p2